ui/src/main/java/org/eclipse/starter/ui/Project.java

```diff
@@ -209,9 +227,6 @@ public void onProfileChange() {
 	}
 
 	public void onJavaVersionChange() {
-		LOGGER.log(Level.INFO,
```

Comment:
Please restore these log entries. They are important to understand what is going on in the application. If log volume is a concern (but quite honestly current log volumes are very modest), the log level can be downgraded a bit.